packages/spruce-skill/interface/pages/skill-views/dashboard_location/index.js

```diff
@@ -111,25 +111,25 @@ class DashboardLocationPage extends React.Component {
 	}
 
 	handleShowConfirmationModal = () => {
-		this.confirm.show(
-			{
+		this.confirm.show({
+			data: {
 				title: 'Are you sure?',
 				text: "Are you sure you want to do that thing you're trying to do?",
 				isDestructive: false,
 				id: Math.random()
 			},
-			eventData => {
+			onConfirm: eventData => {
 				console.log('Confirmed!', eventData)
 			},
-			eventData => {
+			onCancel: eventData => {
 				console.log('Cancelled!', eventData)
 			}
-		)
+		})
 	}
 
 	handleShowConfirmationWithInputModal = () => {
-		this.confirm.show(
-			{
+		this.confirm.show({
+			data: {
```

Comment:
everywhere else i see us putting callbacks (`onConfirm` and `onCancel` on the same level as all other props. Check screen.

https://www.dropbox.com/s/tknfd6wofzrsfkh/Screenshot%202019-06-08%2012.01.03.png?dl=0

So i think we should drop `data` here and do it all on 1 level.

Reply:
updated